server/cli/src/report_utils.rs

```diff
@@ -175,43 +172,31 @@ pub fn generate_report_data(path: &PathBuf) -> Result<ReportData, Error> {
     })
 }
 
-fn run_yarn_install(directory: &PathBuf) -> Result<(), Error> {
-    let convert_dir = directory.join("convert_data_js");
-
-    if !convert_dir.exists() {
-        info!(
-            "No conversion function for {}. Skipping esbuild install.",
-            convert_dir.display().to_string()
-        );
-        return Ok(());
-    }
-
-    let node_modules_path = convert_dir.join("node_modules");
-
-    let package_json = convert_dir.join("package.json");
-    if !package_json.exists() {
-        return Err(Error::FailedToFindPackageJson(convert_dir));
-    }
-
-    let esbuild = convert_dir.join("esbuild.js");
-    if !esbuild.exists() {
-        return Err(Error::FailedToFindEsbuildModule(convert_dir));
-    }
+fn generate_convert_data(path: &PathBuf, manifest: &Manifest) -> Result<Option<String>, Error> {
```

Comment:
just one method to yarn install and convert_data, it's actually faster now without using --check-files and using lock files (but not comiting lock file)